src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/EquippableModule.java

```diff
@@ -0,0 +1,38 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.registrations.Classes;
+import ch.njol.skript.util.ItemSource;
+import ch.njol.skript.util.slot.Slot;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.components.EquippableComponent;
+import org.skriptlang.skript.lang.converter.Converter;
+import org.skriptlang.skript.lang.converter.Converters;
+
+import java.io.IOException;
+
+public class EquippableModule {
```

Comment:
I would consider making this an actual module, even if not using the new syntax registration